### PR TITLE
LPD-23742 Form container in display page templates without update permissions

### PR DESCRIPTION
**Pull request by lfbesada**
https://liferay.atlassian.net/browse/LPD-23742

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by lfbesada** [comment]
ci:test:sf

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java [hunk 1]
```diff
@@ -508,6 +512,117 @@ public void testRenderCollectionStyledLayoutStructureItemWithoutSelectingSegment
 			actualJournalArticle.getArticleId());
 	}
 
+	@Test
+	public void testRenderEditionForm() throws Exception {
+		InfoField<TextInfoFieldType> infoField1 = _getInfoField(false);
+		InfoField<TextInfoFieldType> infoField2 = _getInfoField(false);
+		MockObject mockObject = new MockObject(RandomTestUtil.randomLong());
```
Comment (ealonso) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:519]: I think you should put it at the beginning and add an empty line

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java [hunk 2]
```diff
@@ -508,6 +512,117 @@ public void testRenderCollectionStyledLayoutStructureItemWithoutSelectingSegment
...
+			RenderLayoutStructureTag renderLayoutStructureTag =
+				_getRenderLayoutStructureTagDefaultSegmentsExperience(
+					layout, mockHttpServletRequest, mockHttpServletResponse);
+
+			renderLayoutStructureTag.doTag(
+				mockHttpServletRequest, mockHttpServletResponse);
+
+			String content = mockHttpServletResponse.getContentAsString();
+
+			String fieldsetDisabledHTML = "<fieldset disabled=\"disabled\">";
```
Comment (ealonso) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:558]: You can inline it

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java [hunk 3]
```diff
@@ -508,6 +512,117 @@ public void testRenderCollectionStyledLayoutStructureItemWithoutSelectingSegment
...
+		}
+	}
+
+	@Test
+	public void testRenderEditionFormWithoutUpdatePermission()
+		throws Exception {
+
+		InfoField<TextInfoFieldType> infoField1 = _getInfoField(false);
+		InfoField<TextInfoFieldType> infoField2 = _getInfoField(false);
+		MockObject mockObject = new MockObject(
```
Comment (ealonso) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:575]: Same here

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java [hunk 4]
```diff
@@ -508,6 +512,117 @@ public void testRenderCollectionStyledLayoutStructureItemWithoutSelectingSegment
...
+			RenderLayoutStructureTag renderLayoutStructureTag =
+				_getRenderLayoutStructureTagDefaultSegmentsExperience(
+					layout, mockHttpServletRequest, mockHttpServletResponse);
+
+			renderLayoutStructureTag.doTag(
+				mockHttpServletRequest, mockHttpServletResponse);
+
+			String content = mockHttpServletResponse.getContentAsString();
+
+			String fieldsetDisabledHTML = "<fieldset disabled=\"disabled\">";
```
Comment (ealonso) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/taglib/test/RenderLayoutStructureTagTest.java:615]: Inline

**Pull request closed without merging**